### DISCUSSION TITLE
Project level milestone

### DISCUSSION THREAD

**Discussion by vikash** [community/community#5685] [Projects and Issues]
Right now, the milestones are tied with a repository. But, majority of times we require multiple repositories to work together for a milestone. Having a project level milestone would be very helpful which can be shared across all repos of a project.

**Comment by amacneil** [comment]
This is also very important for us. We are really struggling to use Github issues for tracking milestones, because milestones can only contain tickets from a single repository, and nearly all of our "milestone-worthy" projects touch code in multiple repositories.

We either need the ability for milestones to exist at a project board level, or at a github organization level.

Instead, we have currently resorted to using labels, which is very painful and really not what labels where designed for (and still requires us to create the label in each relevant project).

**Comment by bob6664569** [comment]
Another option would be to visually merge milestones from several repositories, especially useful when grouping lists by a milestone, and this milestone is the same over multiple repositories. But this solution would require more user actions with a double milestone to be set.

**Comment by ghost** [comment]
Duplicate of https://github.com/orgs/github-community/discussions/6296 (as this has more up-votes).
